Sources/BraintreePayPal/Models/PayPalVaultPOSTBody.swift

```diff
@@ -64,29 +54,15 @@ struct PayPalVaultPOSTBody: Encodable {
             self.osType = UIDevice.current.systemName
             self.osVersion = UIDevice.current.systemVersion
             self.universalLink = universalLink.absoluteString
+            self.fallbackURLScheme = fallbackURLScheme
```

Comment:
In the checkout post body we do an `if let` check but we do not for Vault. Are we able to make those consistent?